public-api/db-cdr/generate-cdr/run-achilles-queries.sh

```diff
@@ -46,6 +46,80 @@ fi
 #Get the list of tables in the dataset
 tables=$(bq --project=$BQ_PROJECT --dataset=$BQ_DATASET ls)
 
+################################################
+# CREATE VIEWS
+################################################
+
+# We want to fetch the breakdown counts of each analysis for all of the ehr domain concepts by considering only the records that are ehr related (no ppi results are such)
+# So, instead of joining each of the query with the mapping tables we create the view which are used for ehr only count queries.
+# Mapping tables have the dataset id specified which can be used to differentiate ehr specific rows
+if [[ "$tables" == *"_mapping_"* ]]; then
```

Comment:
In what situations would this not be here? Is this a synthetic vs prod thing?

Reply:
yes exactly, the test/ synthetic data does not have any of the mapping tables.